sdk/docs/sharable/sdk/quickstart/configure/quickstart-project-structure/01-quickstart-rationale.rst

```diff
@@ -1,23 +1,18 @@
 Project overview
 ================
 
-When approaching the Canton Network Quickstart it is important to keep in
-mind that Quickstart is not a "framework" or "platform". Rather, it is a 
-snapshot of an early-stage Canton Network project that has spent several
-sprints creating developer, test, and deployment infrastructure and is
-just beginning to transition to feature delivery.
+The Canton Network (CN) Quickstart is not a "framework" or "platform". 
+It is a snapshot of a CN project that has spent several sprints creating developer, test, and deployment infrastructure and is transitioning to feature delivery.
 
-Developer infrastructure and project logistics are more transferable between
-projects than specific feature delivery. The goal of Canton Network
-Quickstart is to act as a "bootstrap scaffold", to provide the benefits of
-early investment in such project infrastructure to every Canton Network
-Application development team on the network.
+Developer infrastructure and project logistics are more transferable between projects than specific feature delivery. 
+The goal of CN Quickstart is to act as a "bootstrap scaffold", 
+to provide the benefits of early investment in such project infrastructure to every CN Application development team on the network.
 
-As a guide to Quickstart, the treatment of supporting topics is
-necessarily shallow. In addition to the online documentation [1]_,
+As a guide to Quickstart, the treatment of supporting topics is necessarily shallow.
+In addition to the online documentation [1]_,
```

Comment:
Please add links inline.  Andre didn't like doing this and had footnotes.  I don't think that matches the style guide.